src/txmempool.cpp

```diff
@@ -1107,7 +1107,7 @@ CTxMemPool::WriteFeeEstimates(CAutoFile& fileout) const
 {
     try {
         LOCK(cs);
-        fileout << 109900; // version required to read: 0.10.99 or later
+        fileout << 139900; // version required to read: 0.13.99 or later
```

Comment:
This is incorrect, it should be a zcashd client version.

Reply:
Specifically, the first Zcash client version that is allowed to write the new format: v5.5.0-beta1. (In practice the first version that will is v5.5.0-rc1, but using -beta1 makes the version integer nicer.)